core/src/test/java/com/valtech/aem/saas/core/util/UniqueUtilsTest.java

```diff
@@ -0,0 +1,18 @@
+package com.valtech.aem.saas.core.util;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+class UniqueUtilsTest {
+
+  @Test
+  void getUniqueId() {
+    assertThrows(NullPointerException.class, () -> UniqueUtils.getUniqueId(null));
+    assertThat(UniqueUtils.getUniqueId(""), is(""));
+    assertThat(UniqueUtils.getUniqueId("foo"), is("saas_acbd18db"));
```

Comment:
i don't think this test brings any value apart from coverage. we shouldn't care of how the id will look like when testing

Reply:
yes, its soul purpose is coverage, but i will remove it since i'll be using resourcePath.hashCode as you proposed above